go/libkb/secret_store_file.go

```diff
@@ -98,8 +239,36 @@ func (s *SecretStoreFile) ClearSecret(username NormalizedUsername) error {
 	return nil
 }
 
+func (s *SecretStoreFile) clearSecretV2(username NormalizedUsername) error {
+	exists, err := FileExists(s.userpathV2(username))
+	if err != nil {
+		return err
+	}
+	if !exists {
+		return nil
+	}
+
+	for i := 0; i < 3; i++ {
+		noise, err := RandBytes(noiseLen)
+		if err != nil {
+			return err
+		}
+		if err := ioutil.WriteFile(s.noisepathV2(username), noise, 0600); err != nil {
+			return err
+		}
+	}
+
+	if err := os.Remove(s.noisepathV2(username)); err != nil {
+		return err
+	}
+	if err := os.Remove(s.userpathV2(username)); err != nil {
```

Comment:
it probably makes sense to shred this file too -- why not?

So maybe the play is for a `shred()` function that works on a generic file, for the above 3 cases.